app/features/counter/counterSlice.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import { createSlice } from '@reduxjs/toolkit';
+import type { GetState, Dispatch } from '../../reducers/types';
```

Comment:
With TS, we recommend inferring the type of the root reducer as a reusable `RootState` type, and using that with thunks:

https://redux.js.org/recipes/usage-with-typescript#type-checking-reducers

Can that be done with Flow?

Reply:
Yeah this was the main thing I wasn't really sure about as I went through the redux toolkit guide.  I have never used Flow before (only Typescript) so I'm not really sure about this.